src/js/overlay.js

```diff
@@ -319,6 +322,7 @@ Overlay.prototype.broadcast = function(eventType, namespace, detail) {
 	detail = detail || {};
 	detail.el = this;
 
+	console.log(`Firing ${namespace}.${eventType} on ${target}`);
```

Comment:
Could we remove this console message? The template literal makes Travis fail as Travis is running node v0.12.7